src/sources/socket/tcp.rs

```diff
@@ -112,10 +113,10 @@ impl TcpSource for RawTcpSource {
         self.decoder.clone()
     }
 
-    fn handle_events(&self, events: &mut [Event], host: Bytes, byte_size: usize) {
+    fn handle_events(&self, events: &mut [Event], host: Bytes, _byte_size: usize) {
```

Comment:
I'm curious if we can drop the `_byte_size` parameter here now. I'm guessing nothing is using it.